src/main/java/utilities/ServiceUtil.java

```diff
@@ -0,0 +1,89 @@
+package utilities;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+
+public class ServiceUtil {
+
+
+    public Response postResponseFromService(Object requestObject, String url, String projectName, ObjectMapper objectMapper, String token) throws JsonProcessingException {
+        return RestAssured.given().log().all()
+                .header("accept", "application/json")
+                .header("Content-Type", "application/json")
+                .header("Authorization", token)
```

Comment:
Take a look on the .headers(Map<Strint, Object>)
https://www.tutorialspoint.com/how-to-pass-more-than-one-header-in-a-request-in-rest-assured#:~:text=Passing%20them%20as%20a%20Map%20using%20the%20headers%20method.

You will use the same header set for all requests, I guess it make sense to have all of them in one place, and token you can pass as a parameter to the constructor. Is it make sense?

Reply:
Resolved